capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @decoration.correct_name.capitalize!
```

Comment:
- [ ] Using `capitalize!` with the bang (!) returns nil for the method so we are unable to trim the name further. Kindly use just `capitalize` without adding `!`.

  ```
  @decoration.correct_name.capitalize
  ```

  ![mdj](https://user-images.githubusercontent.com/81468675/224271663-2e0efffc-180e-486f-a4f3-2e6ceb01438c.jpg)

Reply:
Hii @julie-ify 
Thanks for suggesting the changes 
I have implemented the following changes